hazelcast-enterprise/Dockerfile

```diff
@@ -58,8 +58,8 @@ RUN echo "Installing new packages" \
     && mv ${HZ_HOME}/tmp/*/* ${HZ_HOME}/ \
     && echo "Setting Pardot ID to 'docker'" \
     && echo 'hazelcastDownloadId=docker' > "${HZ_HOME}/lib/hazelcast-download.properties" \
-    && echo "Granting read permission to ${HZ_HOME}" \
-    && chmod -R +r ${HZ_HOME} \
+    && echo "Granting read&write permission to ${HZ_HOME}" \
+    && chmod -R +rw ${HZ_HOME} \
```

Comment:
`'w'` is not enough to allow files to be written into /opt/hazelcast directory
You have to add `chmod -R a+rwX`